system/otherLoadTest/playlist.xml

```diff
@@ -47,6 +47,21 @@
 			<group>system</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>MiniMix_3h</testCaseName>
+		<variations>
+			<variation>Mode551</variation>
+			<variation>Mode150-CS</variation>
```

Comment:
I doubt that these modes work for Temurin or other hotspot based binaries.  It would be nice to consider a 3h target that can be used for all implementations / vendors also, but certainly will need to verify if `<impls>` tag needed for this PR.

Reply:
> I doubt that these modes work for Temurin or other hotspot based binaries

Yes, these modes should only work on OpenJ9 and IBM impls. I've added `<impl>` tag in the target. 

>  It would be nice to consider a 3h target that can be used for all implementations / vendors 

The original motivation to add this MiniMix_3h target was to be able to have similar coverage we have for an internal (JLTF) test that uses balanced and gencon gc policies. 

May he we should have a separate MiniMix_3h target added for non-IBM implementations? 

@ShelleyLambert